lib/api/model/model.dart

```diff
@@ -205,7 +205,7 @@ class User {
   // bool isOwner; // obsoleted by [role]; ignore
   // bool isAdmin; // obsoleted by [role]; ignore
   // bool isGuest; // obsoleted by [role]; ignore
-  bool? isBillingAdmin; // TODO(server-5)
+  bool isBillingAdmin;
```

Comment:
```
api: Mark isBillingAdmin as requried, relying on server 5+, FL 73+

While `realm_user` update events also have this field added at the same
feature level, it remains optional.

See "Feature level 73" from Zulip API changelog:

   https://zulip.com/api/changelog

Signed-off-by: Zixuan James Li <zixuan@zulip.com>
```

I would just leave out the sentence about how `isBillingAdmin` remains optional in `RealmUserUpdateEvent`. That's the normal, expected thing to happen; it matches all the other `person` fields, in that they might be null if they didn't change in the event.

Also nit in summary line: 'required"